framework/output/LogOutput.kt

```diff
@@ -136,7 +136,7 @@ internal class LogOutput constructor(
                 output(INFO, "...")
                 duration = RUNNING_INDICATOR_DELAY
             } else {
-                duration = RUNNING_INDICATOR_DELAY - Duration.milliseconds(sinceLastResponse)
+                duration = RUNNING_INDICATOR_DELAY - sinceLastResponse.milliseconds
```

Comment:
Let's just quickly make this sound a little less awkward:
```suggestion
                duration = RUNNING_INDICATOR_DELAY - timeSinceLastResponse.milliseconds
```

Reply:
Done.